idaes/dmf/dmfbase.py

```diff
@@ -324,7 +329,7 @@ def _validate_conf(self, c):
             #     'this path, set "{}" in the DMF configuration file.'.format(
```

Comment:
Should these comments be deleted?

Reply:
Yes, you are right. This message should only show up if you try to use the HTML help.

Reply:
@andrewlee94 re: Pitzer data, I agree it should go in this PR and not in a dmf test folder (instead in a data/ directory)